src/index.html

```diff
@@ -11,6 +11,31 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+     action="#"
+     class="search search__big"
+     data-qa="big"
+    >
```

Comment:
Must be 2 spaces everywhere
```suggestion
    <form
      action="#"
      class="search search__big"
      data-qa="big"
    >
```